templates/block/block--companyaddress-2.html.twig

```diff
@@ -0,0 +1,67 @@
+{#
```

Comment:
I am not sure, but why the template name has `2` here ? Does the direct name `templates/block/block--companyaddress.html.twig` not work ?

Reply:
I have checked this but for the company address block, it is showing "block--companyaddress-2.html.twig" in suggestions and it is not taking this twig if i removed 2 from it name.